app/consent-form/_components/Footer.tsx

```diff
@@ -22,7 +22,7 @@ const Footer = () => {
           </Link>
         </div>
       </div>
-      <p className='text-center'>
+      <p className='pb-4 text-center'>
```

Comment:
handle this padding on the footer element where you already have a padding bottom!